src/controllers/topic.controller.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  JsonController,
+  Get,
+  Authorized
+} from 'routing-controllers';
+import { Service } from 'typedi';
+import { TopicsService } from '@services/topics.service';
+import { Topic } from '@entities/topic.entity';
+
+
+@JsonController('/topics')
+@Service()
+export class TopicController {
+  constructor(private readonly topicsService: TopicsService) { }
+
+  @Authorized()
+  @Get()
+  async index(): Promise<Topic[]> {
```

Comment:
I think you could find a more descriptive name for this endpoint, don't you? 

Imagine you are a foreigner and start working on this project and I told you: Pablo, this is the topic controller, what do you think the _index_ function does?

If you struggle to answer this question, then, renaming this could be a good option.